Communityhub/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json

```diff
@@ -1,7 +1,7 @@
 {
-  "downloadLocation": "https://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.11.0.0/secRMMConfigMgrConsoleExtension_9_11_13_0.cab",
-  "FileHash": "548BEB451E606BF287146B4A5000D2CE38711E93D5BF683600A7E7D71E0A35F0",
+  "downloadLocation": "https://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.11.0.0/secRMMConfigMgrConsoleExtension_9_11_14_0.cab",
+  "FileHash": "12C5F25FC81C68B76DC4A7B90449850633DAC3A98540186BB7FBE9DFBA872469",
   "HashAlgorithm": "SHA256",
   "codeSignPolicyFile": "Removable Media Security (secRMM) ConfigMgr console extension-CodeSign-Policy.xml",
-  "Version": "9.11.13.0"
+  "Version": "9.11.14.0"

```

Comment:
>14 [](http://example.com/codeflow?start=19&length=2)

Approving, but please be aware of this.

[https://learn.microsoft.com/en-us/mem/configmgr/core/plan-design/changes/deprecated/removed-and-deprecated-cmfeatures](https://learn.microsoft.com/en-us/mem/configmgr/core/plan-design/changes/deprecated/removed-and-deprecated-cmfeatures)